### PR TITLE
[Notifier] [Telegram] Rework tests

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | --
| License       | MIT

its better to not write json content by hand

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php [hunk 1]
```diff
@@ -94,32 +96,30 @@ public function testSendWithOptions()
...
+                ],
+                'date' => 1459958199,
+                'text' => 'Hello from Bot!',
+            ],
+        ];
 
         $response->expects($this->once())
             ->method('getContent')
-            ->willReturn($content)
+            ->willReturn(json_encode($content))
```
Comment (norkunas) [inline src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php]: maybe would better to use `Symfony\Component\HttpClient\Response\JsonMockResponse` ?

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php]: Like this?

Comment (norkunas) [reply src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php]: Yes :)

**Comment by OskarStark** [comment]
Ready to merge @nicolas-grekas @fabpot

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**